platforms/android/tangram/src/main/java/com/mapzen/tangram/MapController.java

```diff
@@ -1380,9 +1406,19 @@ public void onDrawFrame(final GL10 gl) {
         }
 
         boolean viewComplete;
+        boolean isCameraEasing;
         synchronized(this) {
             viewComplete = nativeUpdate(mapPointer, delta);
-            nativeRender(mapPointer);
+            isCameraEasing = nativeRender(mapPointer);
+        }
+
+        if (isCameraEasing) {
+            uiThreadHandler.post(new Runnable() {
```

Comment:
Small thing: we can make one Runnable and post it repeatedly to avoid excess GC churn.

Reply:
Yeah totally, I feel I have caused this comment at least a couple of times now! Lesson Learnt!